modules/ROOT/partials/configuration/table_responsive_width.adoc

```diff
@@ -1,6 +1,7 @@
+[[table_responsive_width]]
 === `table_responsive_width`
 
-NOTE: This option was deprecated with the release of {productname} 5.4. This option has been replaced by <<table_sizing_mode,`table_sizing_mode`>>. This option will be removed in {productname} 6.0.
+NOTE: This option was deprecated with the release of {productname} 5.4. This option has been replaced by xref:table_sizing_mode[`table_sizing_mode`]. This option will be removed in {productname} 6.0.
```

Comment:
Similar to my last comment this id isn't in this file so this possibly should be this instead:
```suggestion
NOTE: This option was deprecated with the release of {productname} 5.4. This option has been replaced by xref:plugins/opensource/table.adoc#table_sizing_mode[`table_sizing_mode`]. This option will be removed in {productname} 6.0.
```